source/MapPanel.cpp

```diff
@@ -320,9 +331,12 @@ void MapPanel::DrawMiniMap(const PlayerInfo &player, float alpha, const System *
 			RingShader::Draw(to, OUTER, INNER, color);
 		}
 
-		Angle angle;
+		unsigned int missionCounter = 0;
```

Comment:
I believe the style is to use plain `unsigned` over `unsigned int`.

Reply:
Yes, that seems to be the common style; I changed them to `unsigned`.
Can we automatically check for this in the coding style checker? (Which I believe you were working on?)

Reply:
No unfortunately not.